istioctl/cmd/proxyconfig.go

```diff
@@ -813,6 +814,9 @@ func logCmd() *cobra.Command {
 		RunE: func(c *cobra.Command, args []string) error {
 			var err error
 			var loggerNames []string
+			if len(args) > 0 && strings.Contains(args[0], "/ztunnel") {
+				labelSelector = "app=ztunnel"
+			}
```

Comment:
Maybe stupid question - why explicitly reference ztunnel in `istioctl`? What's the special case for `ztunnel` here that wouldn't also apply to other kinds of resources?

Reply:
there are a few spots where we have had to do it already since there will be differences between the ztunnel and envoy based proxies.  A long term solution may be to move the ztunnel commands to their own subcommand rather than under `proxy-config`, `istioctl ztunnel-config <status|workloads|log|secrets|etc>` for instance